packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -452,6 +452,7 @@ export interface CallAdapterCallOperations {
     removeParticipant(participant: CommunicationIdentifier): Promise<void>;
     // @beta
     resumeCall(): Promise<void>;
+    returnToMainMeeting(): Promise<void>;
```

Comment:
I think we need to avoid the term `meeting` throughout (even if this feature today is only available in Teams Meetings). What if instead we have `returnFromBreakoutRoom`?

Reply:
Doesn't solve mainMeetingCallId though... perhaps `originalSessionCallId`, `breakoutRoomOriginCallId` `breakoutRoomReturnCallId` ?

Reply:
cc @dmceachernmsft for ideas too

Reply:
Thanks for the input everyone. I have renamed the function to `returnFromBreakoutRoom` and the property to `breakoutRoomOriginCallId`